website/versioned_docs/version-25.x/WatchPlugins.md

```diff
@@ -157,7 +157,6 @@ For stability and safety reasons, only part of the global configuration keys can
 - [`changedSince`](cli#--changedsince)
 - [`collectCoverage`](configuration#collectcoverage-boolean)
 - [`collectCoverageFrom`](configuration#collectcoveragefrom-array)
-- [`collectCoverageOnlyFrom`](configuration#collectcoverageonlyfrom-array)
```

Comment:
`collectCoverageOnlyFrom` was not documented, so this link was broken in versioned docs.